lintro/config/config_loader.py

```
@@ -302,6 +307,100 @@ def _parse_ai_config(data: dict[str, Any]) -> AIConfig:
+
+    Raises:
+        ValueError: When the checklist item entry is not a mapping.
+    """
+    if not isinstance(data, dict):
+        msg = "review.checklist.items entries must be mappings"
+        raise ValueError(msg)
+
+    known_fields = set(ReviewChecklistItemConfig.model_fields)
+    unknown = set(data) - known_fields
+    if unknown:
+        msg = "Unknown review.checklist.items keys: " f"{', '.join(sorted(unknown))}"
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

(ISC001)